db_scripts/mosip_pms/deploy.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+
```

Comment:
why did we remove the shebang from shell command.

Reply:
in deploy.sh this condition if [ ${DML_FLAG} == 1 ] supported by bash wont support sh
and in all repo's deploy.sh we have not used shebang so removed here also.